channels/offline-channel-js/Tests/Unit/src/offlinetimer.tests.ts

```diff
@@ -43,6 +43,8 @@ export class Offlinetimer extends AITestClass {
                 let channel = new OfflineChannel();
                 let onlineChannel = new TestChannel();
                 this.core.initialize(this.coreConfig,[channel, onlineChannel]);
+
+                this.clock.tick(1);
```

Comment:
I assume that there is already a test which doesn't do this somewhere, so that the PostChannel -> isOffline -> OfflineListener sequence is tested, if not we should probably add that (it will be a very asynchronous test -- and may already be (or be better located) in the e2e tests.

Reply:
added additional test in AISKU with the scenario